QUEST_LV_0100.tsv

```diff
@@ -3823,183 +3823,183 @@ QUEST_LV_0100_20150317_003822	{memo X}Crafting Armor (1)
+QUEST_LV_0100_20150317_003854	$You were asked to inform the troops to assemble before heading to Klaipeda. Inform the Search Scout about the assembly.
+QUEST_LV_0100_20150317_003855	$Inform the Search Scout about the shift.
+QUEST_LV_0100_20150317_003856	$A Large Kepa suddenly appeared. Defeat it.
+QUEST_LV_0100_20150317_003857	$Talk to Search Scout
+QUEST_LV_0100_20150317_003858	$Inform the Search Scout about the order to assemble and find the location of the next soldier.
+QUEST_LV_0100_20150317_003859	$Defeat the Large Kepa
+QUEST_LV_0100_20150317_003860	$Talk to Laimonas
+QUEST_LV_0100_20150317_003861	$Let's listen to Laimonas' favor like Titas said.
+QUEST_LV_0100_20150317_003862	$Offer worship to the Goddess Statue.
+QUEST_LV_0100_20150317_003863	$Offer worship to the Goddess Zemyna Statue. The statue is located at the end of the road on the right of Laimonas.
+QUEST_LV_0100_20150317_003864	$You offered worship like Laimonas said. Tell Laimonas about it.
+QUEST_LV_0100_20150317_003865	$Listen to Laimonas' favor.
+QUEST_LV_0100_20150317_003866	$Defeat the Infrorocktor near the marked area.
+QUEST_LV_0100_20150317_003867	$Do Laimonas a favor and defeat the Infrorocktors on the road to Klaipeda.
+QUEST_LV_0100_20150317_003868	$Talk to the Klaipeda Guard Captain
+QUEST_LV_0100_20150317_003869	$You seem to have taken care of Laimonas' favor. Inform the Guard Captain that you defeated the monsters on the road to Klaipeda.
+QUEST_LV_0100_20150317_003870	$Defeat Infrorocktor
+QUEST_LV_0100_20150317_003871	$Go to Klaipeda
+QUEST_LV_0100_20150317_003872	$Let's move on the way to the Klaipeda.
+QUEST_LV_0100_20150317_003873	$Defeat the Rocktortuga that suddenly appeared.
+QUEST_LV_0100_20150317_003874	$Defeat the Rocktortuga that appeared suddenly before entering the Klaipeda.
+QUEST_LV_0100_20150317_003875	$Report to the Guard Captain on Rocktortuga's defeat.
+QUEST_LV_0100_20150317_003876	$Inform the Battle Commander to return to camp
+QUEST_LV_0100_20150317_003877	$You were asked to tell troops to assemble before going to Klaipeda. Inform the Battle Commander about it.
+QUEST_LV_0100_20150317_003878	$Go to the Hanaming-infested area
+QUEST_LV_0100_20150317_003879	$Go to the Hanaming-infested area and quickly complete the the Battle Commander's assignment.
+QUEST_LV_0100_20150317_003880	$Chase the Hanamings
+QUEST_LV_0100_20150317_003881	$Defeat the surrounding Hanamings
+QUEST_LV_0100_20150317_003882	$Go to the Hanaming habitat
+QUEST_LV_0100_20150317_003883	$Go to the Hanaming habitat and quickly complete the Battle Commander's assignment.
+QUEST_LV_0100_20150317_003884	$Collect Hanaming Petals to investigate.
+QUEST_LV_0100_20150317_003885	$Collect petals from Hanamings to complete the Battle Commander's assignment.
+QUEST_LV_0100_20150317_003886	$Give the Hanaming Petals to the Battle Commander
+QUEST_LV_0100_20150317_003887	$Return to the Battle Commander and give him the Hanaming Petals, so you can be relieved to go to Klaipeda.
```

Comment:
It is possible to shorten "to go to Klaipeda." by replacing it with "to enter Klaipeda."